pkg/virt-handler/virtwrap/schema.go

```diff
@@ -163,6 +169,33 @@ type DiskSourceHost struct {
 
 // END Disk -----------------------------
 
+// BEGIN Serial -----------------------------
+
+type Serial struct {
+	Type         string       `xml:"type,attr"`
+	SerialTarget SerialTarget `xml:"serialTarget"`
+}
+
+type SerialTarget struct {
+	Port string `xml:"port,attr"`
+}
+
+// END Serial -----------------------------
+
+// BEGIN Console -----------------------------
+
+type Console struct {
+	Type          string        `xml:"type,attr"`
+	ConsoleTarget ConsoleTarget `xml:"consoleTarget"`
```

Comment:
I think the xml tag needs to be `xml:"target"`.

Also consider renaming the field name to`Target`.